reference/clinic.v1.yaml

```diff
@@ -303,18 +303,23 @@ paths:
           in: query
           name: period
           description: 'Time Period to display, filter, and sort'
-        - schema:
-            type: boolean
-          in: query
-          name: offsetPeriods
-          required: false
-          description: 'If we should display, filter, and sort based on the offset periods or default periods'
         - schema:
             type: string
             format: date-time
           in: query
           name: lastReviewed
           description: Inclusive
+
+        - schema:
+            $ref: '#/components/schemas/FloatFilter'
+          in: query
+          name: cgm.max
+          description: 'Max glucose value in Mmol/L'
```

Comment:
mmol/L

The case on these matters. mmol/L => millimoles per liter whereas Mmol/L => megamoles per liter.

It's not likely anyone would mistake the two here, but it's always better to be precise.

There are several of these in this review, but I'm only calling out this one.